pallets/did/src/lib.rs

```diff
@@ -87,7 +87,44 @@ mod mock;
 #[cfg(any(feature = "runtime-benchmarks", test))]
 mod mock_utils;
 #[cfg(test)]
-mod tests;
+mod tests{
```

Comment:
No need to have tests defined again here. 
Tests are in `tests.rs`. Can add a new test there.